src/ccalls.jl

```diff
@@ -80,54 +80,56 @@ end
 
 function NEW_MACFLOAT(x::Float64)
     o = ccall( :NEW_MACFLOAT,
-               MPtr,
+               Any,
                (Cdouble,),
                x )
     return o
 end
 
-function ValueMacFloat(x::MPtr)
+function ValueMacFloat(x::GapObj)
     o = ccall( :GAP_ValueMacFloat,
                Cdouble,
-               (MPtr,),
+               (Any,),
                x )
     return o
 end
 
 function CharWithValue(x::Cuchar)
     o = ccall( :GAP_CharWithValue,
-               MPtr,
+               Any,
                (Cuchar,),
                x )
     return o
 end
 
-function ElmList(x::MPtr,position)
+function ElmList(x::GapObj,position)
     o = ccall( :GAP_ElmList,
                Ptr{Cvoid},
-               (MPtr,Culong),
+               (Any,Culong),
                x,Culong(position))
     return GET_FROM_GAP(o)
 end
 
 function NewJuliaFunc(x::Function)
     o = ccall( :NewJuliaFunc,
-               MPtr,
+               Any,
                (Any,),
                x )
     return o
 end
 
 """
-    (func::MPtr)(args...)
+    (func::GapObj)(args...)
 
-> This function makes it possible to call MPtr objects as functions.
+> This function makes it possible to call GapObjs as functions.
 > There is no argument number checking here, all checks on the arguments
 > are done by GAP itself.
 """
-(func::MPtr)(args...; kwargs...) = call_gap_func(func, args...; kwargs...)
+# This needs to be instantiated for `MPtr` and is therefore moved to the init function
```

Comment:
Which init function is that? Could you clarify?

Reply:
Well, the one below the comment, but I can make this more specific